frontend/src/component/photo/edit/details.vue

```diff
@@ -671,6 +680,41 @@ export default {
         }
       }
     },
+    onCoordinateInputChange(value) {
+      this.coordinateInput = value;
+
+      if (this.isValidCoordinateInput) {
+        this.applyCoordinates();
+      }
+    },
+    applyCoordinates() {
+      if (!this.isValidCoordinateInput) return;
+
+      const parts = this.coordinateInput.split(",").map((part) => part.trim());
+      const lat = parseFloat(parts[0]);
+      const lng = parseFloat(parts[1]);
+
+      // Update underlying model
+      this.view.model.Lat = lat;
+      this.view.model.Lng = lng;
+      this.view.model.PlaceSrc = "manual";
+    },
+    clearLocation() {
+      this.view.model.Lat = 0;
+      this.view.model.Lng = 0;
+      this.view.model.PlaceSrc = "manual";
+      this.updateCoordinateInput();
+    },
+    updateCoordinateInput() {
+      const lat = this.view.model.Lat;
+      const lng = this.view.model.Lng;
+
+      if (lat !== null && lng !== null && !(lat === 0 && lng === 0) && !isNaN(lat) && !isNaN(lng)) {
+        this.coordinateInput = `${parseFloat(lat).toFixed(6)}, ${parseFloat(lng).toFixed(6)}`;
```

Comment:
Note that we need to be careful with rounding because it can cause pictures to be placed incorrectly on the map, depending on the specific latitude and longitude. Shortening the coordinates in the input must not affect the data sent to the backend. In any case, the original data should be preserved if no changes are made?